packages/atomic/src/components/facets-v1/facet-search/facet-search.pcss

```diff
@@ -0,0 +1,27 @@
+input[type="text"]:focus ~ .search-icon {
+    @apply text-primary;
+}
+
+/**
+ * @prop --atomic-facet-search-ring-color: Color of the input's outline ring
```

Comment:
Those are atomically parsed by Stencils Doc

Reply:
What does that annotation generate as far as documentation is concerned ?

Reply:
In the generated JSON, for the new facet:

![Screen Shot 2021-06-07 at 11 39 02 AM](https://user-images.githubusercontent.com/4923043/121048626-f67e1280-c784-11eb-86d9-82e714b8ac84.png)